Dockerfile

```diff
@@ -15,21 +15,21 @@
 FROM registry.access.redhat.com/ubi8/openjdk-8:1.3-8
 
 # Build parameters
-ARG IQ_SERVER_VERSION=1.116.0-01
-ARG IQ_SERVER_SHA256=4463213916bd935bb0c9f33fbcf59b6af0d7fc13a362ba0cb2041a1be30c6330
+ARG IQ_SERVER_VERSION=1.117.0-01
```

Comment:
Do I need to manually set these to higher version?

Reply:
> Do I need to manually set these to higher version?

The version and hashes are automatically updated during a release
https://github.com/sonatype/docker-nexus-iq-server/commit/54f0ee0b2c51afdf865b898c636bd36b3accfd0c

I think something might be wrong with the rebase it shouldn't be showing this as a change since it's already 1.117.0-01 on master